clients/client-python/build.gradle.kts

```diff
@@ -167,6 +167,11 @@ tasks {
     args = listOf("scripts/generate_version.py")
   }
 
+  val doc by registering(VenvTask::class) {
```

Comment:
I think it would be better to change the task name to `pydoc`, what do you think? I found that in Java we use javadoc for task name, maybe we should align with Java.

Reply:
Sure, and it also align with our other tasks like `pylint` and `black`, also named by its executable.